frontend/src/components/UserDetail/UserDetail.tsx

```diff
@@ -97,14 +97,14 @@ const UserDetail = ({
       });
     }
   };
+  if (isShowing && rider) {
+    showToast(
+      `Rider ${rider.active ? 'activated' : 'deactivated'}.`,
+      toastType ? ToastStatus.SUCCESS : ToastStatus.ERROR
```

Comment:
Since it has been changed to use the enum only lets refactor the toggleActive functionality to behave like this 

set the success and error in the then and catch clauses respectively 
So no need for an if statement 
showToast can be called directly in the then and catch clause.